.dockerignore

```diff
@@ -0,0 +1,3 @@
+.git
+node_modules
+build
```

Comment:
It a minor thing, but there should be a newline at the end of each file.

Reply:
@hankwallace  will make sure to add a newline at end of any file.